scripts/produceLKG.ts

```diff
@@ -15,10 +15,10 @@ async function produceLKG() {
     await copyLibFiles();
     await copyLocalizedDiagnostics();
     await copyTypesMap();
-    await buildProtocol();
     await copyScriptOutputs();
     await copyDeclarationOutputs();
     await writeGitAttributes();
+    await buildProtocol();

```

Comment:
I would swap these two lines just to group these closer by relevance.

```suggestion
    await buildProtocol();
    await writeGitAttributes();
```

Reply:
Neither of those two truly have any relevance to the rest, their order doesn't matter, relatively speaking~ All the others are just grouped `copy`s.